### PR TITLE
Create post

### PR DESCRIPTION
**Pull request by foolhardy21**
## Create Post Component.

This PR has APIs and UI components related to creating a new post.

- Files to review:
   - [ ] src/components/Reusable/CreatePost.jsx
   - [ ] src/contexts/posts.context.js
   - [ ] src/reducers/posts.reducer.js
   - [ ] src/pages/Explore.jsx

- Use this link for preview - https://deploy-preview-8--remarkable-flan-74a807.netlify.app/login

v-1

Thanks.

### CODE REVIEW

File: src/components/Reusable/CreatePost.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            const createdPost = response.data.posts[response.data.posts.length - 1]
+            postsDispatch({ type: ACTION_ADD_POST, payload: createdPost })
+        } else if (response.status === 404) {
+            // not logged in
+        }
+        setPostValue('')
+    }
+
+    return (
+        <article className={`${postStyles.postDiv} flx flx-column pd-s`}>
```
Comment (ApurvaSawant11) [inline src/components/Reusable/CreatePost.jsx:25]: Good that you are using semantic HTML

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! You can merge the PR

**Review by anik31 — APPROVED** [review]
Everything looks good. Code is clean & you have broken down the code into pieces which is good.

**Review by Hamza-777 — APPROVED** [review]
Looks good

**Pull request merged**